listenbrainz/spark/handlers.py

```diff
@@ -197,10 +197,20 @@ def handle_dump_imported(data):
 
     dump_name = data['imported_dump']
     import_completion_time = data['time']
+
+    if dump_name:
+        template = 'emails/dump_import_notification.txt'
+        subject = 'A {} has been imported into the Spark cluster'.format(' '.join(data['type'].split('_')[1:]))
+        recipients = ['listenbrainz-observability@metabrainz.org']
+    else:
+        template = 'emails/dump_import_failure.txt'
+        subject = 'No dump imported into Spark cluster!'
```

Comment:
If there is any way we can make this statement more useful (download failed, no file available, hash failed), that would be excellent.